PlayCover/Views/MenuBarView.swift

```diff
@@ -93,7 +94,16 @@ struct PlayCoverViewMenuView: Commands {
         }
         CommandGroup(before: .sidebar) {
             Button("menubar.clearCache") {
-                ImageCache.clearCache()
+                DataCache.instance.cleanAll()
+                 URLCache.iconCache.removeAllCachedResponses()
+                 do {
+                     let cacheFolder = FileManager.default.urls(for: .cachesDirectory, in: .userDomainMask).first!
+                      if FileManager.default.fileExists(atPath: cacheFolder.path) {
+                          try FileManager.default.removeItem(at: cacheFolder)
+                      }
+                  } catch {
+                      Log.shared.error(error)
+                  }
```

Comment:
Wrong indentation

Reply:
Elaborate please. This is the old cache folder for reference

Reply:
These lines have one more space than other lines

Reply:
Oh! I see